### PR TITLE
Added link to the chrome webstore

### PR DESCRIPTION
**Pull request by Explosion-Scratch**
^^^

**Resolves**

Resolves #

**Changes**

**Reason for changes**

**Tests**

### CODE REVIEW

**Comment by GrahamSH-LLK** [comment]
What about firefox??

**Comment by Explosion-Scratch** [comment]
> What about firefox??

Done!

File: README.md [hunk 1]
```diff
@@ -30,11 +30,11 @@ Scratch Addons by itself is just an addon loader. Its main tasks are:
...
 
 ## Installation
-### From the chrome webstore
+### From the Chrome webstore
 Simply go to [the Scratch Addons extension](https://chrome.google.com/webstore/detail/scratch-addons/fbeffbjdlemaoicjdapfpikkikjoneco) in the Google Chrome Webstore and click ```add extension``` and follow the on-screen steps.
-### From the FireFox webstore
+### From the Firefox webstore
 Go to the extension [in the Firefox webstore](https://addons.mozilla.org/en-US/firefox/addon/scratch-messaging-extension/) and click 'Add to FireFox'. Congrats! You're all set, enjoy ScratchAddons!
-### Download and instal locally
+### Download and install locally
```
Comment (Explosion-Scratch) [inline README.md]: Thanks!

Comment (Explosion-Scratch) [inline README.md]: I thought I checked for typos though...

Comment (Explosion-Scratch) [reply README.md]: Oh well lol

**Comment by WorldLanguages** [comment]
I believe @Hans5958 added something like this to the readme branch?

**Comment by Hans5958** [comment]
Have you seen https://github.com/ScratchAddons/ScratchAddons/tree/readme?
I also need to add another section later on the Install section.

**Comment by GrahamSH-LLK** [comment]
@Hans5958 has already created something like this, plus some other things, so I think we'll merge that.

**Comment by Hans5958** [comment]
I'm sorry to tell you, but due to my mistake on the end, I accidentally force-pushed into the master branch, closing this pull request as the result. I have reversed the changes, but I can't reopen the pull request. I'm not sure if this is still needed, but you need to make another pull request if it is.

Once again, I apologize for the inconvenience that caused from my mistake.

**Comment by Explosion-Scratch** [comment]
It's fine!

On Friday, October 2, 2020, Hans5958 <notifications@github.com> wrote:
> I'm sorry to tell you, but due to my mistake on the end, I accidentally
force-pushed into the master branch, closing this pull request as the
result. I have reversed the changes, but I can't reopen the pull request.
I'm not sure if this is still needed, but you need to make another pull
request if it is.
>
> Once again, I apologize for the inconvenience that caused from my mistake.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.<
https://ci5.googleusercontent.com/proxy/HokhuBf005lGW-VC7aoqbn0zKqlIcqu_6P0bzwfmrYxkGqF4AK5oxmNRhLwH7xjxhjLyrwbUnIiqbTqvSK_5fl70jB5UWeCy7kn6X3I5kCnalrPvymB68-Nl3mL334BWQOUUqy8BtXbyX5ecxzObLEZDUo8BsfriD5T1DCzytenWK4Z5WsQBjmg2lv3a5cOXEBpvYrOFdXPsY5SFunS92wGAIWfNhg_NoEkGJbt8qw=s0-d-e1-ft#https://github.com/notifications/beacon/AOT2P3TQ4J4XM5TRO62ILV3SIVIFHA5CNFSM4R7OLHC2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOFHPY5JY.gif>

**Pull request closed without merging**